File: 7kyu/isDivisibleBy.py
# Create a function isDivisible(n,...) that checks if the first argument n is divisible by all other arguments
# (return true if no other arguments)

def is_divisible(*args):
    first = args[0]
    for i in range(1, len(args)):
        if first % args[i] != 0:
            return False
    return True

File: 7kyu/isDivisibleBy.js
// Create a function isDivisible(n,...) that checks if the first argument n is divisible by all other arguments
// (return true if no other arguments)

function isDivisible(){

  for (var i = 0; i < arguments.length; i++){
    if (arguments[0] % arguments[i] !== 0){
      return false;
    }
  }
  return true;
}


